### DISCUSSION TITLE
Media/App keys?

### DISCUSSION THREAD

**Discussion by ghost** [cajhin/capsicain#99] [General]
Hi, I just got a new keyboard (Vermilo VA108) which has 4 keys that launch applications, I'm trying to figure out if there's a way I can map those to do volume up or volume down. I couldn't find anything about it while I was scouring through the docs, but maybe I missed something?

**Comment by cajhin** [comment]
see https://github.com/cajhin/capsicain/wiki/X-doesn%27t-work
